### ISSUE TITLE
Agent: Requests should be claimed only once

### ISSUE THREAD

**Issue by GabrielBuragev** [beamer-bridge/beamer#1319]
#### Expected behavior
Agents should claim (call `claimRequest` method) the requests only once.

#### Actual behavior
Agents are claiming (calling `claimRequest` method) the same request multiple times.

#### Steps to reproduce the behavior
- Create a request on any rollup
- Agent claims the request once after filling it on destination rollup
- Restart the agent
- Now, the agent claims the same request again

**Comment by fredo** [comment]
can you reproduce this behavior on Goerli with the latest Agent version (the one we run from source currently) between Optimism and Arbitrum (excluding Boba)?

**Comment by GabrielBuragev** [comment]
Yeah, you can see the last `Claim Request` transactions here: https://goerli-optimism.etherscan.io/address/0x9d3F5EE65BF588C89d690272D0984774Fb77590A

**Comment by istankovic** [comment]
We should have a test for this in our test suite.
